examples/apps/src/ble_l2cap_peripheral.rs

```diff
@@ -86,3 +86,15 @@ where
     })
     .await;
 }
+
+// General Q:
+//  What is the purpose of L2CAP in the examples?  I get that there is a 'bas' example, but what
+//  would be the case where a user of 'TrouBLE' wants to deal with the (did I get it right?) lower
+//  level of abstraction?
+//
+//  What may make this problematic is the layout having both 'ble' and 'l2cap' alongside each other,
+//  as siblings. When one likely is a higher abstraction; 'bas' using 'l2cap' underneath (just
+//  assuming; please correct me or point to a learning resource?).
+
```

Comment:
In BLE, you can use l2cap connection oriented channels (as done in the examples), completely without gatt. GATT is a protocol that is built on top of a single l2cap channel, so it's a different level of abstraction, which includes all of the standardized GATT characteristics etc.

Using l2cap connection oriented channels is nice if you just want to send messages between a peripheral and central. It might be useful to think of it as a UDP (though with more reliability guarantees and flow control).